Products/GenericSetup/registry.py

```diff
@@ -162,7 +163,7 @@ def endElement(self, name):
             deps = tuple(self._pending['dependencies'])
             self._pending['dependencies'] = deps
 
-            desc = ''.join(self._pending['description'])
+            desc = str().join(self._pending['description'])
```

Comment:
That doesn't smell right -- the literal is much more idiomatic, and there can't be any difference in meaning.

Reply:
Huh, no idea what I was thinking.